### PR TITLE
#160577472 Add coveralls coverage badge

### PR DESCRIPTION
**Pull request by ibetween**
### What does this PR do?

- Integrates Coveralls code coverage service
### Description of Task to be completed?
- [ah-shakas](https://github.com/andela/ah-shakas) Repo has been integrated with coveralls code coverage service 
- Coveralls code coverage service will ensure that we are properly informed just how well our code base is tested which in turn provides quality assurance for the product.
- The affected file is only the [Readme.md](https://github.com/andela/ah-shakas/blob/ch-integrate-coveralls-160577472/README.md) whereby a code coverage badge has been added.
### How should this be manually tested?
- The integration status can be viewed [here](https://coveralls.io/github/andela/ah-shakas)
- Coverage for your the next Travis build should appear here 👉 [Coverage Status](https://coveralls.io/github/andela/ah-shakas?branch=develop)

### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/160577472

### CODE REVIEW

**Comment by codingedward** [comment]
I have gone through the implementation and I believe this is ready for merging.

**Review by codingedward — COMMENTED** [review]
I have gone through the implementation and I believe this is ready for merging.

**Review by codingedward — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Status
+
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-shakas/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-shakas?branch=develop)
```
Comment (FawazFarid) [inline README.md]: can you change this to master? I'm gonna explain a bit later why

**Review by FawazFarid — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ibetween) [reply README.md]: Yeah, sure.

Comment (ibetween) [reply README.md]: @FawazFarid I have made the proposed changes.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# Status
+
+[![Coverage Status](https://coveralls.io/repos/github/andela/ah-shakas/badge.svg?branch=develop)](https://coveralls.io/github/andela/ah-shakas?branch=master)
```
Comment (FawazFarid) [inline README.md]: You've only changed the second line, what about the first? It's still develop

**Review by FawazFarid — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ibetween) [reply README.md]: 🤔 I see. Correcting right away.

Comment (ibetween) [reply README.md]: Hi @FawazFarid. I have made the changes.

**Review by FawazFarid — APPROVED** [review]

**Pull request merged**